exercises/CI/introduction_to_ci/README.md

```diff
@@ -0,0 +1,154 @@
+  +     xjTXjgNPCaVej50967MDn0kvrIGy6/X6Bk7A+0A+WaPqMcjNHbFjGDFUsc4X5x2r
+  +     4FUd+8nKghGvuK1AnKzhRFNcFIDjHYZ6ooSv2yDZBH2aiWDRmek7GdsCgYEAzbSV
+  +     wemxCusW/GhdOaSrd7+Z1NwU3wubb/reiVy+lHjSRJ93uEkGreUvn2cfrQPMIr4J
+  +     z1IVGGt1Vpg0FAkJ+cJfppo+akzuP0Nfzi7PnXlUqWUVPdRfhHyqc2ypyIFr2Y2I
+  +     Y9gIwaDCWrTXMPYasl4Y1nvtONeDG9l6y76dx9ECgYARBs63V+NU5Ais+eoY5Fig
+  +     cRxJo3+N9i1B4jmxOJdQrvE6nPzX9X85GYXMyD5CFdy5deRTAW9HMvq3s3KdL8/1
+  +     kJey649tABfa0Hw7jZmynf2hRxQzb8YDJOuqGS6cUk9FOA5A3Wv2LzlD1Jf//ted
+  +     bFI1m9yEi5yx5qW1bKRJ8QKBgQCqwiSrExYfjSXIo833ouWgA1Jsq8E7ooQGwgbn
+  +     mjCuSVmrE4GrXhuIUT9mIJHf6s4TbRPxoR7X5aRDHRfmJda9GQiT4Wi8j14a8A/g
+  +     tWUtQOHxuHKYn+omreJE7MhIJH9l2iKFV3l1WsDb0VDtWO6ycIBH1BoJXIV+odtY
+  +     tMR/cQKBgE2L/lEzFR+/d80d39ECGGaCllC2F2cGz/BSWNle5Cp7PVs8y23yG3HG
+  +     iOCmPStUUZ7hi8OVjfsszJqv/z/Tp6kx/hNHXoPFmGcD4G6JyiifwlGG4tLMsO+g
+  +     SJm2WgmccOTu2RVpUkBcPKgxENQWO85QnTF9MlVjjuGRelWcrXwi
+  +     -----END RSA PRIVATE KEY-----
+  +   uri: ssh://git@git-server/git-server/repos/application-repo.git
+  
+jobs:
+  job my-first-job has been added:
+  + name: my-first-job
+  + plan:
+  + - do:
+  +   - aggregate:
+  +     - get: repo
+  +       trigger: true
+  +   - task: pointless-echo
+  +     config:
+  +       platform: linux
+  +       image_resource:
+  +         type: docker-image
+  +         source:
+  +           repository: alpine
+  +       run:
+  +         path: sh
+  +         args:
+  +         - -e
+  +         - -c
+  +         - -x
+  +         - |
+  +           echo "Something worked"
+  +       inputs:
+  +       - name: repo
+  
+pipeline created!
+you can view your pipeline here: 127.0.0.1:8080/teams/main/pipelines/ci-intro-pipeline
+
+the pipeline is currently paused. to unpause, either:
+  - run the unpause-pipeline command
+  - click play next to the pipeline in the web ui
+```
+
+
+
+Visit the [concourse home page](http://127.0.0.1:8080) again and you'll see the pipeline that the was just pushed.
+![Pending Pipeline](./pending_pipeline_screenshot.png)
+
+
+You'll notice the following:
+- there is now a pipe line called: `ci-intro-pipeline`
+- the pipeline is paused
+
+When pipelines are first pushed, they are not enabled. In order to run the pipeline we need to unpause it. Do this by either clicking the play symbol or by running: `./resources/linux/fly -t local unpause-pipeline -p ci-intro-pipeline`. Unpausing the pipeline should cause Concourse to trigger a build for our new pipeline. Go back to the [detailed view of the pipeline](127.0.0.1:8080/teams/main/pipelines/ci-intro-pipeline) after 20 seconds or so, you'll notice that that pipeline goes green. This means that the code has been pulled successfully from GIT and that the singe job, that is in the pipeline, has executed successfully.
+
+![Passing Pipeline](./passing_pipeline_screenshot.png)
```

Comment:
In my case, concourse is unable to connect to the git server, so pipeline never goes green for me.